src/treemodel/document.js

```diff
@@ -307,16 +311,15 @@ export default class Document {
 	 * * 'removeRootAttribute' when attribute for root is removed,
 	 * * 'changeRootAttribute' when attribute for root changes.
 	 *
-	 * Change event is fired after the change is done. This means that any ranges or positions passed in
```

Comment:
Because it was not true - `data.range` is indeed after the change, but `data.position` is before the change. I added additional note in params comments.